packages/cli/src/utils.ts

```diff
@@ -192,3 +210,15 @@ export function getCodeContentType(filename: string): string {
   }
   return ContentType.TEXT;
 }
+
+export function createProfile(profileName: string, options: any): void {
+  const storage = new FileSystemStorage(profileName);
+  const optionsObject = { name: profileName, ...options };
+  storage.setObject('options', optionsObject);
+  console.log(`${profileName} profile created`);
+}
+
+export function getProfileOptions(profileName: string): any {
```

Comment:
See comment above - let's use consistent naming strategy here with `loadProfile` or `readProfile`

Also, let's use the `Profile` for return type rather than `any`